### PR TITLE
Do not display unsafe exception message

### PR DESCRIPTION
**Pull request by rn0**

### CODE REVIEW

File: Resources/views/Security/login.html.twig [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <form class="form-signin" role="form" action="{{ path("fsi_admin_security_user_check") }}" method="post">
         {% if error %}
-        <div class="alert alert-danger">{{ error.message|trans({}, 'FSiAdminSecurity') }}</div>
+        <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
```
Comment (szymach) [inline Resources/views/Security/login.html.twig]: This `security` translation domain is from Symfony?

**Comment by szymach** [comment]
Also what is the purpose of this PR?

**Comment by rn0** [comment]
@szymach it is from Symfony

This PR aim to solve insecure displaying of full exception message with potential sensitive information.

**Comment by JarekW** [comment]
what about `FSiAdminSecurity.en.yml` and translation domain?

**Comment by szymach** [comment]
@rn0 rebase

**Comment by szymach** [comment]
Closed in favour of #84

**Pull request closed without merging**